modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/ActionButton/index.js

```diff
@@ -108,15 +110,38 @@ const ActionButton = ({
 		productName
 	);
 
-	return (
-		<ButtonDropDown
-			items={activationKeysActionsItems}
-			label={i18n.translate('actions')}
-			menuElementAttrs={{
-				className: 'p-0',
-			}}
-		/>
+	const filteredKeysActionsItems = getFilteredKeysActionsItems(
+		project?.accountKey,
+		provisioningServerAPI,
+		sessionId,
+		handleAlertStatus,
+		handleRedirectPage,
+		handleDeactivatePage,
+		productName
 	);
+
+	if (isAdminOrPartnerManager === true) {
```

Comment:
Oi Luisa, já que isAdmin é um boolean, não precisa do === true.
```suggestion
	if (isAdminOrPartnerManager) {
```